internal/urlutil_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright (C)  2018 Yahoo Japan Corporation Athenz team.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package urlutil contains the utility functions for URL processing
+package urlutil
```

Comment:
we should name it url instead of urlutil. no util package name is allowed in golang. 
Also this class should put in internal/url folder instead of internal folder.

Reply:
There is `https://golang.org/pkg/io/ioutil/`. Others just said `util` package is not good as they are too ambiguous.

`internal/url` or `internal/urlutil`?
As we are working on string, not URL exactly...

Reply:
renamed